SEED Benchmark/OEI/classes/pollseedforproperties.xml

```diff
@@ -99,106 +114,141 @@ return sdf.parse(sdf.format(c.getTime()));
+                      value="#[flowVars.ErrorLog + &quot;\n \n [Error1] SEED property: &quot; + flowVars.PropertyDetails.state.property_name + &quot;, SEED Id: &quot; + flowVars.PropertyId + &quot;. Neither '&quot; + flowVars.CompliedStatus + &quot;' or '&quot; + flowVars.ViolationStatus + &quot;' labels have been applied to this property so no action will be taken.&quot; ]"
+                      doc:name="Variable: Error Log"/>
+      </otherwise>
+    </choice>
+    <catch-exception-strategy doc:name="pollseedforpropertiesCatch_Exception_Strategy">
+      <set-variable variableName="ErrorLog"
+                    value="#[flowVars.ErrorLog + &quot;\n \n [Error4] SEED Label: '&quot; + flowVars.LabelID + &quot;' assigned to Property: &quot; + flowVars.PropertyName + &quot; has no details in SEED. &quot;]"
+                    doc:name="Variable"/>
+      <set-variable variableName="ErrorLog"
+                    value="#[flowVars.ErrorLog + &quot;.  \n As a result the SEED property named: [(&quot;  + flowVars.PropertyName + &quot;),  Seed ID: (&quot; + flowVars.PropertyId + &quot;)] will not be updated in Salesforce until the issue is resolved.  Confirm that the unavailable SEED label is removed from this property.&quot;]"
+                    doc:name="Variable"/>
+    </catch-exception-strategy>
+  </flow>
+  <sub-flow name="QuerySFforContactAndAccount">
+    <set-variable variableName="contactEmail"
+                  value="#[flowVars.PropertyDetails.state.extra_data[&quot;Email&quot;].trim()]"
+                  doc:name="Variable: Contact Email useful for getting SF Account"/>
+    <set-variable variableName="ContactName"
+                  value="#[flowVars.PropertyDetails.state.extra_data[&quot;On Behalf Of&quot;]]"
+                  doc:name="Variable: Contact Name"/>
+    <set-payload value=" #[flowVars.PropertyDetails.state.extra_data.Email.trim()]" doc:name="Set Payload"/>
+
+    <sfdc:query-single config-ref="Testbed-Salesforce"
+                       query="Select Id, Email, Account.Name from Contact where Email = '#[flowVars.contactEmail]'"
+                       doc:name="Salesforce: Call for Contact and Account details"/>
+    <choice doc:name="Choice">
+      <when expression="#[payload != null]">
+        <set-variable variableName="ContactResults" value="#[payload.Id]" doc:name="Variable: Contact details"/>
+      </when>
+      <otherwise>
+        <flow-ref name="CreateContactAndAccount" doc:name="CreateContactAndAccount"/>
+      </otherwise>
+    </choice>
+  </sub-flow>
+  <flow name="CreateContactAndAccount">
+    <set-variable variableName="AccountName"
+                  value="#[flowVars.PropertyDetails.state.extra_data[&quot;Organization&quot;].trim()]"
+                  doc:name="Variable: SF Account Name"/>
+    <sfdc:query-single config-ref="Testbed-Salesforce"
+                       query="Select Id from Account where Name = '#[flowVars.AccountName.replace("'", "\\'")]'"

```

Comment:
This is failing XML validation. 

`SEED Benchmark/OEI/classes/pollseedforproperties.xml: Failed to xml parse (SEED Benchmark/OEI/classes/pollseedforproperties.xml:202:99: not well-formed (invalid token))`

Is this the right syntax?

Reply:
And just to confirm... Mule doesn't like it either. Here is the mule log

```
Caused by: org.mule.api.config.ConfigurationException: Line 202 in XML document from URL [file:/opt/mule-standalone-3.8.0/apps/oei/pollseedforproperties.xml] is invalid; nested exception is org.xml.sax.SAXParseException: Element type "sfdc:query-single" must be followed by either attribute specifications, ">" or "/>". (org.mule.api.lifecycle.InitialisationException)
```